keyboards/donutcables/budget96/keymaps/vial/rules.mk

```
@@ -0,0 +1,27 @@
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+LTO_ENABLE = yes
+QMK_SETTINGS = no
+
+MAGIC_ENABLE = no
+BOOTMAGIC_ENABLE = no
+MOUSEKEY_ENABLE = no
+CONSOLE_ENABLE = no
+COMMAND_ENABLE = no
+COMBO_ENABLE = no
+NKRO_ENABLE = no
+RING_BUFFERED_6KRO_REPORT_ENABLE = no
+AUDIO_ENABLE = no
+KEY_OVERRIDE_ENABLE = no
+
+LEADER_ENABLE = no
+MIDI_ENABLE = no
+UNICODE_ENABLE = no
+BUETOOTH_ENABLE = no
+RAW_ENABLE = no
```

```suggestion
MOUSEKEY_ENABLE = no
CONSOLE_ENABLE = no
COMMAND_ENABLE = no
COMBO_ENABLE = no
NKRO_ENABLE = no
KEY_OVERRIDE_ENABLE = no
```
Almost all of this isn't enabled by default (and `BLUETOOTH_ENABLE` is misspelled)
`VIA_ENABLE = yes` forcibly enables both `BOOTMAGIC` and `RAW`, so no point trying to set them to `no`

Though ultimately, having junk in your Vial rules.mk doesn't matter since it'll just be ignored.